packages/react/src/internal/components/TextInputInnerAction.tsx

```diff
@@ -92,13 +101,28 @@ const TextInputAction = forwardRef<HTMLButtonElement, TextInputActionProps>(
       console.warn('Use the `aria-label` prop to provide an accessible label for assistive technology')
     }
 
+    const accessibleLabel = ariaLabel
+      ? {'aria-label': ariaLabel}
```

Comment:
This gives priority to aria-label over aria-labelledby which is opposite of what browsers do. Is that intentional?

> aria-labelledby takes precedence over all other methods of providing an accessible name, including aria-label
Ref: https://arc.net/l/quote/ksytdmqp

Reply:
Yes, kind of. 

Because `aria-label` is a prop on the `TextInputInnerAction` component and it is used to provide tooltip text as well. 

If there is `aria-labelledby`, the tooltip wiring needs to be manually. So to my understanding, `aria-label` is encouraged and preferred. 

Let me know if you have any concerns or anything I am missing here.

Reply:
I see!